src/app/rosetta/lib/construction.ml

```diff
@@ -290,8 +293,10 @@ module Metadata = struct
           Metadata_data.create ~sender:options.Options.sender
             ~token_id:options.Options.token_id ~nonce
           |> Metadata_data.to_yojson
-      ; suggested_fee= [Amount_of.coda (Unsigned.UInt64.of_int 1_000_000_000)]
-      }
+      ; suggested_fee=
+          [ Amount_of.coda
+              (MinaCurrency.Fee.to_uint64
+                 Mina_compile_config.default_transaction_fee) ] }
```

Comment:
Did you mean to add `minimum_user_command_fee` (as per the PR description)?

Reply:
Thanks.  As far as I understand, this is to be done in a separate PR as it would not be a hotfix (see #8086).